Makefile

```diff
@@ -290,6 +308,36 @@ clean:
 	@rm -f $(LIB_DIR)/mpxe-gen/src/*.pb-c.c
 	@rm -f $(MPXE_DIR)/protogen/*_pb2.py
 
+# Note: build.py relies on a lot of relative paths so it would be easier to just cd and execute command 
+.PHONY: codegen
+codegen: codegen_protos
+	@echo "Generating from templates..."
+	@cd $(CODEGEN_DIR) && python scripts/build.py 
+	@find $(CODEGEN_DIR)/out -type f \( -iname '*.[ch]' -o -iname '*.ts' \) | xargs -r clang-format -i -fallback-style=Google
+	@find $(CODEGEN_DIR)/out -type f \( -iname '*.go'  \) | xargs -r gofmt -w
+	@find $(CODEGEN_DIR)/out -name \*.h -exec cp {} libraries/codegen-tooling/inc/ \;
+
+# Note: build_dbc has same issue as build.by with local paths
+.PHONY: codegen_dbc
+codegen_dbc:
```

Comment:
MPXE should probably be updated to depend on this and use the new dbc file location, but that's for another PR (no action requested for this comment). @jessm